model/client4.go

```diff
@@ -1035,10 +1039,23 @@ func (c *Client4) CreateUserAccessToken(userId, description string) (*UserAccess
 	}
 }
 
-// GetUserAccessToken will get a user access token's id, description and the user_id
-// of the user it is for. The actual token will not be returned. Must have the
-// 'read_user_access_token' permission and if getting for another user, must have the
-// 'edit_other_users' permission.
+// GetAllUserAccessToken will get a all access token's id, description, is_active
+// and the user_id in the system. The actual token will not be returned. Must have
+// the 'manage_system' permission.
+func (c *Client4) GetAllUserAccessTokens(page int, perPage int) ([]*UserAccessToken, *Response) {
```

Comment:
Might want to remove the 'All' bit from here if it's removed from the endpoint

Reply:
I renamed this to `GetUserAccessTokens` and have changed the previous definition of `getUserAccessTokens` to `getUserAccessTokensForUser` so I can use `getUserAccessTokens` and so the codebase is consistent.